src/main/java/com/jcabi/email/Protocol.java

```diff
@@ -124,7 +124,9 @@ public SMTPS(final String hst, final int prt) {
         @Override
         public Map<String, String> entries() {
             return new ImmutableMap.Builder<String, String>()
-                .putAll(new Protocol.SMTP(this.host, this.port).entries())
+                .put("mail.smtps.auth", Boolean.TRUE.toString())
+                .put("mail.smtps.host", this.host)
+                .put("mail.smtps.port", Integer.toString(this.port))
```

Comment:
Properties from Protocol.SMPT were not ok since the ``smtps`` transport used in the SMTPS wire looks up the properties with ".smtps." in the middle, not ".smtp.". If it does not find them, the defaults are used (localhost and port 465)